target/linux/lantiq/dts/RS353.dts

```diff
@@ -0,0 +1,318 @@
+/dts-v1/;
+
+/include/ "vr9.dtsi"
+
+/ {
+	model = "RS353 - Bintec Elmeg RS353";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200 init=/etc/preinit";
+		leds {
+                        running = &status;
+                        boot = &status;
+			wifi = &wlan;
+			usb2 = &usb; /* Front port is mapped to device 2 */
+			dsl = &dsl;
+                };
+
+	};
+
+	memory@0 {
+		reg = <0x0 0x8000000>;
+	};
+
+	sram@1F000000 {
+		cgu0: cgu@103000 {
+                        compatible = "lantiq,cgu-xway";
+                        reg = <0x103000 0x1000>;
+                        #clock-cells = <1>;
+			lantiq,phy-clk-src = <0x2>;
+		};
+	};
+
+	fpi@10000000 {
+		localbus@0 {
+			nor-boot@0 {
+				compatible = "lantiq,nor";
+				bank-width = <2>;
+				reg = <0 0x0 0x2000000>;
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				partitions {
+					compatible = "fixed-partitions";
+					#address-cells = <1>;
+					#size-cells = <1>;
+
+					boardconfig: partition@0 {
```

Comment:
It seems the bit 30 controls the EBU endianness swap, as you can see [here](https://github.com/openwrt-vgv7519/openocd-lantiq/blob/master/target/arx100.cfg):

```
proc arx100_ebu_swap_enable {} {
    echo "ARX100: EBU swap enable"

    mww 0xbe105360 0x4001f7ff
}

proc arx100_ebu_swap_disable {} {
    echo "ARX100: EBU swap disable"

    mww 0xbe105360 0x0001f7ff
}
```

It seems that some platforms doesn't need the EBU bus swapping. In the kernel sources, we can see the EBU swapping needs to be activated to make the PCI bus working. But in my case, the PCI (PCIe exactly) works just fine with or without bus swapping.